stl/inc/sstream

```diff
@@ -457,29 +458,27 @@ protected:
 
 #if _HAS_CXX20
     void _Init_string_inplace(_Mystr&& _Str, int _State) {
-        _State |= _From_rvalue;
+        if ((_State & _Noread) && (_State & _Constant)) { // Cannot read / write buffer, do nothing
+            _Seekhigh = nullptr;
+            _Mystate  = _State | _From_rvalue;

```

Comment:
I am slightly unsure whether the `_From_rvalue` is really applicable here

Reply:
Seems reasonable to keep.